Application/EdFi.Ods.Features/OpenApiMetadata/Factories/OpenApiMetadataDefinitionsFactory.cs

```diff
@@ -60,35 +60,31 @@ private static Schema EtagSchema
                 .Where(x => _openApiMetadataFactoryResourceFilterStrategy.ShouldInclude(x.Resource))
                 .SelectMany(
                     r => new[]
-                    {
-                        new
                         {
-                            key = _openApiMetadataDefinitionsFactoryNamingStrategy.GetResourceName(r.Resource, r),
-                            schema = CreateResourceSchema(r)
-                        },
-                        isChangeQueriesEnabled
-                            ? new
-                            {
-                                key = GetTrackedChangesResourceKeyName(r.Resource, r),
-                                schema = CreateTrackedChangesKeySchema(r)
-                            }
-                            : null,
-                        isChangeQueriesEnabled
-                            ? new
-                            {
-                                key = GetTrackedChangesResourceDeleteName(r.Resource, r),
-                                schema = CreateTrackedChangesDeletesSchema(r)
-                            }
-                            : null,
-                        isChangeQueriesEnabled
-                            ? new
-                            {
-                                key = GetTrackedChangesResourceKeyChangeName(r.Resource, r),
-                                schema = CreateTrackedChangesKeyChangesSchema(r)
-                            }
-                            : null
-                    })
-                .Where(d => d != null)
+                            (
+                                key: _openApiMetadataDefinitionsFactoryNamingStrategy.GetResourceName(r.Resource, r),
+                                schema: CreateResourceSchema(r)
+                            )
+                        }
+                        .Concat(
+                            isChangeQueriesEnabled && !r.IsCompositeResource
```

Comment:
This is just a slight improvement, don't include Change Queries schemas for Composite since they are not needed. Before this change, the definitions were included but never used.